Dockerfile

```diff
@@ -12,15 +12,14 @@ ADD setup.py /opt/atomicapp/
 ADD requirements.txt /opt/atomicapp/
 
 WORKDIR /opt/atomicapp
-
-RUN pip install .
+RUN python setup.py install
```

Comment:
There is lot a of confusion on which Dockerfile to use.
After trying out all possible options , I found that pip install doesnt seem to install dependecies in requirement.txt and also we need /usr/bin/atomicapp which we can get via python setup.py install
IS there a reason to revert this ?